### PR TITLE
Shorten the crc name

### PR DESCRIPTION
**Pull request by hongkailiu**
Follow up https://coreos.slack.com/archives/C018MHR6SHM/p1598630674007100?thread_ts=1598628980.003400&cid=C018MHR6SHM

```
len(99-pd-53728114-21bf-4af7-a0bc-3fe4ee908d6d-containerruntime)=59
```

/cc @alvaroaleman @umohnani8

### CODE REVIEW

**Comment by umohnani8** [comment]
LGTM

File: clusters/build-clusters/02_cluster/openshift-machine-api/pthread-debug_mcp.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-  name: pthread-debug
+  name: pd
   labels:
-    pthread-debug: "true"
+    pd: "true"
 spec:
   machineConfigSelector:
     matchExpressions:
-      - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker,pthread-debug]}
+      - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker,pd]}
```
Comment (alvaroaleman) [inline clusters/build-clusters/02_cluster/openshift-machine-api/pthread-debug_mcp.yaml:10]: what? We only want that for the pthread instance so the rest remains without this fix so we can test that the issue does occur on the non-fix instances and does not occur on the fix instances
/hold

**Comment by hongkailiu** [comment]
I thought this PR is merged already. Let me remove the last commit.

File: clusters/build-clusters/02_cluster/openshift-machine-api/pthread-debug_mcp.yaml (hunk 1 above)
Comment (hongkailiu) [reply clusters/build-clusters/02_cluster/openshift-machine-api/pthread-debug_mcp.yaml]: I understood this.
I need to make the `containerruntimeconfig` take effects (`pidsLimit: 0`) first, right?

**Comment by hongkailiu** [comment]
```
oc --context build02 delete mcp pthread-debug --as system:admin
machineconfigpool.machineconfiguration.openshift.io "pthread-debug" deleted
oc --context build02 delete containerruntimeconfig pthread-debug --as system:admin
containerruntimeconfig.machineconfiguration.openshift.io "pthread-debug" deleted
```

**Pull request merged**